ProjFS.Mac/PrjFSKextTests/PrjFSKextTests.exp

```diff
@@ -0,0 +1,6 @@
+# The explicit exported symbol list allows dead code stripping to work on the
+# test bundle binary.
+
+_OBJC_CLASS_$*
+_OBJC_METACLASS_$*
+
```

Comment:
@pmj, what error(s) were you seeing without this file?

Reply:
As the comment suggests, dead code stripping does nothing unless we explicitly specify the exported symbols. What's happening is:

 - The test target is a "bundle" executable. This is normally used for plugins etc. - essentially, the test bundle is dynamically loaded into the test runner.
 - Dynamic libraries export all symbols with external linkage by default, i.e. any functions not explicitly marked as having hidden or local (`static`) linkage.
 - This means dead code stripping can't really strip anything, as all functions are exported by default.
 - Many functions in KauthHandler.cpp in turn depend on various KPI functions and other function in the kext. These aren't (yet) defined in the test target bundle, so they produce link errors.

Exporting only the Objective-C classes (test case classes) means only the functions called by test methods survive dead code stripping, so we don't immediately depend on the world, only on the true dependencies of the tested functions.

Reply:
Ahh that makes sense, thanks!